docs/pages/machine-id/getting-started.mdx

```diff
@@ -102,7 +102,7 @@ Machine ID can join with a token or the [IAM Method](https://goteleport.com/docs
 
     # Define the name of the bot that will be allowed to use this token. Note,
     # the "bot-" prefix is required. The name of the bot you will create is "robot".
```

Comment:
Need to update the comment here. It still says `bot-` is required.

Reply:
Whoops, I passed right over that. Fixed!